pkg/plugins/external/init.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package external
 
 import (
+	"github.com/spf13/pflag"
 	"sigs.k8s.io/kubebuilder/v3/pkg/machinery"
```

Comment:
```suggestion

	"sigs.k8s.io/kubebuilder/v3/pkg/machinery"
```

nit space to sort the imports

Reply:
Should be done for all imports as of commit https://github.com/kubernetes-sigs/kubebuilder/pull/2693/commits/9648e34decca255094c2a4b1934d92ae014caa78